next.config.js

```diff
@@ -38,5 +38,20 @@ module.exports = withHashicorp({
       domains: ['www.datocms-assets.com'],
       disableStaticImages: true,
     },
+    webpack(config) {
+      // disable `exports` support for framer-motion. This will cause Webpack to
+      // fall back to using `module`
+      config.module.rules.push({
+        test: (input) => {
+          return (
+            input.includes('framer-motion') || input.includes('motion-config')
+          )
+        },
+        resolve: {
+          exportsFields: [],
+        },
+      })
+      return config
+    },
```

Comment:
This should be included in `v2.1.0` of `@hashicorp/platform-nextjs-plugin`

Reply:
@BRKalow yeah we need another release to handle the `react-motion-config -> framer-motion` resolution chain. This was a temporary workaround to unblock swingset previews :)

Reply:
https://github.com/hashicorp/web-platform-packages/pull/66